<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1540471958640" stop="1540472350250" version="1.4.23" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default Suite : Zetwerk</name>
    <title>Default Suite : Zetwerk</title>
    <test-cases>
        <test-case start="1540471969584" stop="1540472006372" status="passed">
            <name>verify_Add_Supplier_Button[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472013136" stop="1540472045956" status="passed">
            <name>verify_Add_Supplier_CompanyName_ErrorMessage[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472054150" stop="1540472086966" status="passed">
            <name>verify_Add_Supplier_ContactNumber_ErrorMessage[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472093258" stop="1540472125960" status="passed">
            <name>verify_Add_Supplier_EmailID_ErrorMessage[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472133262" stop="1540472166154" status="passed">
            <name>verify_Add_Supplier_FirstName_ErrorMessage[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472174492" stop="1540472207223" status="passed">
            <name>verify_Add_Supplier_Location_ErrorMessage[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472213911" stop="1540472231485" status="failed">
            <name>verify_FavouriteTab_When_NoFavouriteAdded[hunbun1469@gmail.com,377b853b]</name>
            <title>Verify Favourite tab when there is no favourite added</title>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.pers_aip.Zetwerk.SuppliersPageTest.SuppliersPageTest.verify_FavouriteTab_When_NoFavouriteAdded(SuppliersPageTest.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:200)
	at com.pers_aip.TestBase.run(TestBase.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:212)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments>
                <attachment title="Failure in method verify_FavouriteTab_When_NoFavouriteAdded" source="ca048a0f-acc7-41c5-8d04-a4fc6ceff1ef-attachment.png" type="image/png"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472238424" stop="1540472255743" status="failed">
            <name>verify_Find_NewSuppliersTab[hunbun1469@gmail.com,377b853b]</name>
            <title>Verify Find New Suppliers Tab</title>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.pers_aip.Zetwerk.SuppliersPageTest.SuppliersPageTest.verify_Find_NewSuppliersTab(SuppliersPageTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:200)
	at com.pers_aip.TestBase.run(TestBase.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:212)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments>
                <attachment title="Failure in method verify_Find_NewSuppliersTab" source="f818bd87-59b9-42dc-a9ab-a3513c26603c-attachment.png" type="image/png"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472262615" stop="1540472286470" status="passed">
            <name>verify_SearchBox_Functionaloity_underSuppliers[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472292663" stop="1540472310818" status="failed">
            <name>verify_SuppliersButton[hunbun1469@gmail.com,377b853b]</name>
            <title>Verify whether Supplier Tab is launched Properly or Not</title>
            <failure>
                <message>AssertionError: expected [true] but found [false]</message>
                <stack-trace>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.pers_aip.Zetwerk.SuppliersPageTest.SuppliersPageTest.verify_SuppliersButton(SuppliersPageTest.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:200)
	at com.pers_aip.TestBase.run(TestBase.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:212)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments>
                <attachment title="Failure in method verify_SuppliersButton" source="cf3c3d47-3a68-453a-bcb8-7f3257cd047b-attachment.png" type="image/png"/>
            </attachments>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1540472317314" stop="1540472350061" status="passed">
            <name>verify_TotalNoOf_Suppliers[hunbun1469@gmail.com,377b853b]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-6P0RF6P"/>
                <label name="thread" value="17496@DESKTOP-6P0RF6P.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
